src/main/resources/resources/vocabulary/sql/conceptSetQuery.sql

```diff
@@ -1 +1 @@
-select concept_id from @vocabulary_database_schema.CONCEPT where concept_id in (@conceptIds)and invalid_reason is null
+select concept_id from @vocabulary_database_schema.CONCEPT where concept_id in (@conceptIds)
```

Comment:
It's a bit concerning. I don't believe we want to select invalid concepts in all cases.
Probably it's better to make this optional.

Reply:
I have the same concerns here, but there was a thread on this at forums.ohdsi.org and they insist that the conceptSetQuery should return invalid concepts.  My fear is that we'll have a lot more results in the query, but this is what was decided (also, see #35 for details)